configure.seed

```diff
@@ -13,10 +13,10 @@ AS_IF([test "${with_sanitizer+set}" = set],[
 
 LT_INIT
 
-AC_PROG_CC(clang gcc)
-AM_PROG_CC_C_O(clang gcc)
-AC_PROG_CXX(clang++ g++)
-AC_PROG_CC_STDC(clang gcc)
+AC_PROG_CC
+AM_PROG_CC_C_O
+AC_PROG_CXX
+AC_PROG_CC_STDC
```

Comment:
Doing this change, on my Mac clang won't be used. I don't think it's a good idea to revert the change. What do you think?

Reply:
Strange. Can you post/upload `config.log` w/ and w/o those changes?
Would help me a lot.

Reply:
Attached

Reply:
Thanks, will investigate.

Reply:
keeping my fingers crossed that it will work now